frontend/public/src/components/Header.js

```diff
@@ -27,11 +29,20 @@ const Header = ({ currentUser, location }: Props) => {
       scope.setUser(null)
     })
   }
-  return (
-    <React.Fragment>
-      <TopAppBar currentUser={currentUser} location={location} />
-    </React.Fragment>
-  )
+  const showNewDesign = checkFeatureFlag("mitxonline-new-header")
```

Comment:
Spent some time debugging this with Jenni, as I noticed this still returns true even when local posthog call is disabled after testing the feature flag is on. We think it's cache issue with JS call since other feature flags use python call. It should be fine but let's see on RC